src/poliastro/twobody/orbit.py

```diff
@@ -443,6 +444,50 @@ def from_horizons(
         )
         return ss
 
+    @classmethod
+    def from_sbdb(cls, name, **kargs):
+        """Return osculating `Orbit` by using `SBDB` from Astroquery.
+
+        Parameters
+        ----------
+        body_name: string
+            Name of the body to make the request.
+
+        Returns
+        -------
+        ss: poliastro.twobody.orbit.Orbit
+            Orbit corresponding to body_name
+
+        Examples
+        --------
+        >>> from poliastro.twobody.orbit import Orbit
+        >>> apophis_orbit = Orbit.from_sbdb('apophis')
+        """
+
+        obj = SBDB.query(name, full_precision=True, **kargs)
+
+        a = obj["orbit"]["elements"]["a"].to(u.AU) * u.AU
+        ecc = float(obj["orbit"]["elements"]["e"]) * u.one
+        inc = obj["orbit"]["elements"]["i"].to(u.deg) * u.deg
+        raan = obj["orbit"]["elements"]["om"].to(u.deg) * u.deg
+        argp = obj["orbit"]["elements"]["w"].to(u.deg) * u.deg
+        nu = obj["orbit"]["elements"]["ma"].to(u.deg) * u.deg
```

Comment:
I think this is the mean anomaly!

Reply:
You are completely righ @Juanlu001, I will solve this in a moment. Sorry for the error, I'll pay more attention next time.